src/App.js

```diff
@@ -1,25 +1,46 @@
+import React from 'react'
 import logo from './logo.svg';
 import './App.css';
+import Footer from './components/Footer'
+import Header from './components/Header'
+import CatIndex from './pages/CatIndex'
+import CatShow from './pages/CatShow'
+import CatNew from './pages/CatNew'
+import CatEdit from './pages/CatEdit'
+import Home from './pages/Home'
+import NotFound from './pages/NotFound'
+import { Routes, Route } from "react-router-dom"
+import cats from './mockCats.js'
+// import 'bootstrap/dist/css/bootstrap.min.css';
+// import Navbar from './components/Header'
```

Comment:
Let's remove this commented out code from production

Reply:
removed commented out code from production